src/table/settings/gui_settings.ini

```diff
@@ -130,12 +130,16 @@ def      = false
 str      = STR_CONFIG_SETTING_SMOOTH_SCROLLING
 strhelp  = STR_CONFIG_SETTING_SMOOTH_SCROLLING_HELPTEXT
 
-[SDTC_BOOL]
+[SDTC_VAR]
 var      = gui.right_mouse_wnd_close
-flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC
-def      = false
+type     = SLE_UINT8
+flags    = SF_NOT_IN_SAVE | SF_NO_NETWORK_SYNC | SF_GUI_DROPDOWN
+def      = 0
+min      = 0
+max      = 2
```

Comment:
You might be able to use the enum values here instead of integers, not sure though.

Reply:
Was this not possible?

Reply:
This slipped my mind when I picked this back up, this is possible and has been done in the latest commit